.NET/Microsoft.Recognizers.Text.DataDrivenTests/NumberWithUnit/TestNumberWithUnit_Chs.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Generic;
+
+namespace Microsoft.Recognizers.Text.DataDrivenTests.NumberWithUnit
+{
+    [TestClass]
+    public class TestNumberWithUnit_Chs : TestBase
+    {
+        public static TestResources TestResources { get; protected set; }
+        public static IDictionary<string, IModel> Models { get; protected set; }
+
+        [ClassInitialize]
+        public static void ClassInitialize(TestContext context)
+        {
+            TestResources = new TestResources();
+            TestResources.InitFromTestContext(context);
+            Models = new Dictionary<string, IModel>();
+        }
+
+        [TestInitialize]
+        public void TestInitialize()
+        {
+            base.TestSpecInitialize(TestResources);
+            base.ModelInitialize(Models);
+        }
+
+        [DataSource("Microsoft.VisualStudio.TestTools.DataSource.CSV", "AgeModel-Chs.csv", "AgeModel-Chs#csv", DataAccessMethod.Sequential)]
+        [TestMethod]
+        public void AgeModel()
+        {
+            base.TestNumberWithUnit();
+        }
+
+        [DataSource("Microsoft.VisualStudio.TestTools.DataSource.CSV", "CurrencyModel-Chs.csv", "CurrencyModel-Chs#csv", DataAccessMethod.Sequential)]
+        [TestMethod]
+        public void CurrencyModel()
+        {
+            base.TestNumberWithUnit();
+        }
+
+        [DataSource("Microsoft.VisualStudio.TestTools.DataSource.CSV", "DimensionModel-Chs.csv", "DimensionModel-Chs#csv", DataAccessMethod.Sequential)]
+        [TestMethod]
+        public void DimensionModel()
+        {
+            base.TestNumberWithUnit();
+        }
+
+        [DataSource("Microsoft.VisualStudio.TestTools.DataSource.CSV", "TemperatureModel-Chs.csv", "TemperatureModel-Chs#csv", DataAccessMethod.Sequential)]
```

Comment:
I see these are pointing to CSVs, but the PR includes a few jsons.
What's the expected flow for this PR? Just confirming.

Reply:
The DataSource requires a CSV file to work. The TestMethod then iterates on all the CSV rows.
The TestInitialize method loads all test cases from the Spec file and creates a CSV with the same amount of test cases. The CSV is then used only as an iterator for the [TestMethod].

It uses a similar approach as to MsTestHacks, but much more focused to our problem.